webapp/client/src/pages/PauschbetragPage.js

```diff
@@ -95,9 +95,8 @@ export default function PauschbetragPage({
 }
 
 PauschbetragPage.propTypes = {
-  stepHeader: PropTypes.exact({
+  stepHeader: PropTypes.shape({
```

Comment:
Why did you change that?

Reply:
The stepHeader.intro will be ignored from the frontend. When the backend provides the data, it includes the Intro field with a None value. The validation of the PropTypes would then fail because you use .exact() instead of .shape().

Reply:
Thanks for explaining that

Reply:
Why shouldn't the intro then just be optional? Wouldn't that make it more explicit?

Reply:
Should we define it like this:

```
PauschbetragPage.propTypes = {
  stepHeader: PropTypes.exact({
              title: PropTypes.string.isRequired,
              intro: PropTypes.string,
}
```

Proptypes still want a structure in a exact scheme and we define something that we don't use at all. The right way should be:

```
PauschbetragPage.propTypes = {
  stepHeader: PropTypes.shape({
              title: PropTypes.string.isRequired,
              intro: PropTypes.string,
}
```

And here I ask myself if we really need intro defined here because we don't use it in the page. What are your thoughts?

Reply:
Honestly, I think the intro just shouldn't be set if we don't use it? Let's huddle over this^^